### DISCUSSION TITLE
Usage with Remix

### DISCUSSION THREAD

**Discussion by lukebars** [ascorbic/unpic-img#170] [Q&A] [answered]
Does `unpic-img` support Remix on Vercel? What should be the suggested setup?

**Comment by ascorbic** [comment]
Hi. I've not tried it, but it should work fine. Just use the regular `@unpic/react` component. Let me know how you get on

**Comment by danestves** [comment, marked as answer]
I use @unpic/react without any issues on Remix, works like a charm tbh

**Reply by lukebars** [reply]
@danestves I'm experiencing an issue with local images though, it says "Invalid URL" for anything that I try to grab locally, yet anything from remote works like a charm. Does it work fine for you? 🤔

**Reply by danestves** [reply]
Oh Unpic is made for remote images not local as I remember 👀

**Reply by lukebars** [reply]
Oh, I guess local images might be Next.js only feature. As it's only mentioned in the Next.js part 🤔

**Reply by ascorbic** [reply]
For local images with Next.js it just uses the next/image handler
